src/System.Private.ServiceModel/src/System/IdentityModel/Claims/WindowsClaimSet.cs

```diff
@@ -1,16 +1,349 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using System.IdentityModel.Policy;
 using System.Collections.Generic;
-using System.Collections.ObjectModel;
-using System.ComponentModel;
+using System.IdentityModel.Policy;
 using System.Runtime;
-using System.Runtime.InteropServices;
 using System.Security;
 using System.Security.Principal;
 using System.ServiceModel;
 
 namespace System.IdentityModel.Claims
 {
+    public class WindowsClaimSet : ClaimSet, IIdentityInfo, IDisposable
+    {
+        internal const bool DefaultIncludeWindowsGroups = true;
+#if FEATURE_NETNATIVE //NegotiateStream
```

Comment:
Could we have 2 files for this? One called WindowsClaimSet.NetNative.cs and the other WindowsClaimSet.CoreClr.cs? Having basically two compete implementations of this class in a single file doesn't seem like the right approach.